src/Dashboard/Model.purs

```diff
@@ -21,6 +21,12 @@ type CommitRow =
   , commitTitle :: String
   }
 
+type NameRow =
+  { id          :: Gitlab.ProjectId
+  , name        :: Gitlab.ProjectName
+  , name_with_namespace   :: Gitlab.ProjectNameWithNamespace
+  }
```

Comment:
The aligning here should either be fixed or removed. I'd say better follow smart advice and [don't align code](https://github.com/ianbollinger/purescript-style-guide/blob/master/purescript-style.md#19-dont-align-code)

Reply:
As the test of the code is aligned I feel aligning is the way to go, we could unalign the whole project in a separate commit later if we stop aligning code as a rule.